packages/calling-stateful-client/src/MediaAccessSubscriber.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { MediaAccessCallFeature, MediaAccessChangedEvent } from '@azure/communication-calling';
```

Comment:
For stable builds to build without error, the imports will also need to be conditional-compiled.
To test stable builds, you can do:
```sh
rush switch-flavor:stable
```
then build the packages as normal, sometime `rush build` will surpress the build errors and just say `error`, if that happens `cd` into the directory and run `rushx build`

```sh
cd packages/calling-stateful-client
rush build -T . # handy command to build packages before this directory (only needs run once if oyu have just switched flavor)
rushx build # build the current dir
```

Reply:
Good catch, thanks :)

Reply:
Pushed fix.